pytest_tests/testsuites/object/test_object_lock.py

```diff
@@ -119,6 +123,15 @@ def locked_storage_object(
 
 @pytest.mark.grpc_object_lock
 class TestObjectLockWithGrpc(ClusterTestBase):
+
```

Comment:
do we need an empty line here? seems like it was not here before

Reply:
> do we need an empty line here? seems like it was not here before

Fixed